src/api/config/routes.rb

```diff
@@ -164,11 +164,10 @@ def self.matches?(request)
     end
 
     controller 'webui/project' do
-      get 'project/' => :index
-      get 'project/list_public' => :list_public
-      get 'project/list_all' => :list_all
+      get 'project/' => :index, as: 'projects'
+      get 'project/list_public' => :index
+      get 'project/list_all' => :index, show_all: true
```

Comment:
Well we are not exactly in need to test config/routes.rb right? So if you are visiting project/list_all and it goes to index with the parameter show_all then we are fine, right?